ci/cron/daily-compat.yml

```diff
@@ -258,4 +264,20 @@ jobs:
         displayName: 'Blackduck Scan'
         env:
           BLACKDUCK_HUBDETECT_TOKEN: $(BLACKDUCK_HUBDETECT_TOKEN)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          source $(bash_lib)
+
+          tr -d '\015' <*_Black_Duck_Notices_Report.txt | grep -v dach-ny_daml-on-corda >NOTICES
+          if git diff --exit-code -- NOTICES; then
```

Comment:
@garyverhaegen-da i think the git diff bombed.  i did the more crude dry run git add because that seemed to work more reliably on a big diff

```/tmp/tmp.O5s1Eh6hgk: line 18: branch: unbound variable

##[error]Bash exited with code '1'.
```